src/structures/game_session.ts

```diff
@@ -357,6 +357,14 @@ export default class GameSession {
         this.prepareRound(randomSong.name, randomSong.artist, randomSong.youtubeLink);
         this.gameRound.setBaseExpReward(await this.calculateBaseExp(guildPreference));
 
+        if ((await checkBotIsAlone(this, this.voiceChannel))) {
+            return;
+        }
+        if (this.voiceChannel.voiceMembers.size === 0) {
```

Comment:
Isn't this redundant

Reply:
This handles the case where before any rounds start the user leaves the voice channel. checkBotIsAlone only ends the session if the bot is the only one the voice channel (members == 1)